nextcloud/firstrunwizard
# [stable10] Fix firstrunwizard

* backport of #16 
* cc @karlitschek this fixes a "white page after login in IE and Edge" - see nextcloud/server#1620


How to test:
* create a new user
* enable first run wizard
* log in with this new user via IE or Edge
* before: file list is empty (completely white without "nothing in this folder" message)
* after: first run wizard is shown and file list is not empty

cc @nickvergessen @rullzer @klm46
